src/bio/bio_buffer.c

```diff
@@ -763,11 +763,9 @@ void
 bio_memcpy(struct bio_desc *biod, uint16_t media, void *media_addr,
 	   void *addr, ssize_t n)
 {
-	struct umem_instance *umem = biod->bd_ctxt->bic_umem;
-
 	if (biod->bd_update && media == DAOS_MEDIA_SCM) {
-		pmemobj_memcpy_persist(umem->umm_pool, media_addr,
-				       addr, n);
+		/* NB: pmemobj_tx_commit will drain HW buffer */
+		pmem_memcpy_nodrain(media_addr, addr, n);
```

Comment:
I think we'd better call pmemobj API like pmemobj_memcpy() with PMEMOBJ_F_MEM_NODRAIN flag (it could be different with the pmem_memcpy_nodrain() when the pmemobj pool is replicated)